### PR TITLE
Added a precision

### PR DESCRIPTION
**Pull request by hervehobbes**

### CODE REVIEW

File: en/appendices/4-0-upgrade-guide.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ plugin:
     bin/cake upgrade file_rename templates <path/to/app>
 
 Once you've renamed your template and locale files, make sure you update
-``App.paths.locales`` and ``App.paths.templates`` paths to be correct.
+``App.paths.locales`` and ``App.paths.templates`` (in ``/config/app.php``) paths to be correct.
```
Comment (markstory) [inline en/appendices/4-0-upgrade-guide.rst]:
```suggestion
``App.paths.locales`` and ``App.paths.templates`` (in **/config/app.php**) paths to be correct.
```

Comment (hervehobbes) [reply en/appendices/4-0-upgrade-guide.rst]: Excellent suggestion, thank you.

Comment (dereuromark) [reply en/appendices/4-0-upgrade-guide.rst]: I would put the () part behind the word paths.

Comment (othercorey) [inline en/appendices/4-0-upgrade-guide.rst]:
```suggestion
``App.paths.locales`` and ``App.paths.templates`` paths (in **/config/app.php**) to be correct.
```

Comment (othercorey) [reply en/appendices/4-0-upgrade-guide.rst]: Updated to be after `paths`. I cannot commit this fix to merge though.

Comment (dereuromark) [reply en/appendices/4-0-upgrade-guide.rst:55]: why are we using bold here? I think the code fencing is cleaner.

Comment (markstory) [reply en/appendices/4-0-upgrade-guide.rst:55]: Paths are bold generally in the docs. inline code is for classes, variables and methods.

**Comment by dereuromark** [comment]
Are we ok to move forward here?

**Comment by markstory** [comment]
Thank you 👍

**Pull request merged**